README.md

```diff
@@ -1,2 +1,11 @@
 # gpii-qa-automation
 Repository for the GPII QA automation code
+# GPII UI Automation
+UI Automation is done with Java 8 and the used tool is Sikuli. More information here -> http://sikulix.com/quickstart/ .
+Currently automation code is executed only on Windows but Sikuli is cross platform tool so when the GPII App becane cross platform automation - test cases could be executed on Mac and Linux distribution.
+### Running the Tests in a VM
```

Comment:
Please itemise any further 3rd-party dependencies in addition to Sikuli's, e.g. Maven, together with version requirements and sketch or pointer to appropriate install instructions